### PR TITLE
Update mocks to be in sync with backend responses

### PR DESCRIPTION
**Pull request by gowthamidommety-okta**
Resolves: OKTA-394650

## Description:

Backend JIRA - https://oktainc.atlassian.net/browse/OKTA-394674

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- [OKTA-394650](https://oktainc.atlassian.net/browse/OKTA-394650)

### CODE REVIEW

File: playground/mocks/data/idp/idx/error-okta-verify-totp.json [hunk 1]
```diff
@@ -117,9 +117,6 @@
   "currentAuthenticator": {
     "type": "object",
     "value": {
-      "profile": {
-        "deviceName": "OnePlus 7 Pro"
-      },
```
Comment (jmelberg-okta) [inline playground/mocks/data/idp/idx/error-okta-verify-totp.json:122]: Do we know when this API change was made? If we can, let's reference the ticket here.

**Review by jmelberg-okta — APPROVED** [review]
Thanks for doing this!

File: playground/mocks/data/idp/idx/error-okta-verify-totp.json (hunk 1 above)
Comment (gowthamidommety-okta) [reply playground/mocks/data/idp/idx/error-okta-verify-totp.json:122]: Sure, will add that 👍 API change was just made today

File: playground/mocks/data/idp/idx/error-authenticator-webauthn-failure.json [hunk 2]
```diff
@@ -175,7 +175,7 @@
       "type":"security_key",
       "key":"webauthn",
       "id":"aut12uqz8PnlDpzQ20g4",
-      "displayName":"MacBook Touch ID",
+      "displayName":"Security Key or Biometric Authenticator",
```
Comment (SamSanjabi-okta) [inline playground/mocks/data/idp/idx/error-authenticator-webauthn-failure.json:178]: I think you added "Authenticator" by mistake to this one.

**Review by SamSanjabi-okta — APPROVED** [review]
Think you accidentally added "Authenticator" to one of the strings, but otherwise everything looks good.

File: playground/mocks/data/idp/idx/error-authenticator-webauthn-failure.json (hunk 2 above)
Comment (gowthamidommety-okta) [reply playground/mocks/data/idp/idx/error-authenticator-webauthn-failure.json:178]: Thank you!

**Pull request closed without merging**